woocommerce/storefront
# Use 'wp-scripts test-e2e' instead of directly calling jest

Supersedes #1958 and it better aligns Storefront e2e tests with what we have in WC Blocks.

### How to test the changes in this Pull Request:

1. Make sure e2e tests pass in this PR.
2. Optionally, verify tests also pass locally:
2.1. Start docker.
2.2. Start wp-env: `npm run wp-end start`.
2.3. Build Storefront: `npm run build`.
2.4. Run tests: `npm run test:e2e`.
2.5. Verify all tests are a success.
